shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1568,12 +1848,15 @@ - (void)resetAllClientIds {
 }
 
 - (void)addToInputParentViewIfNeeded:(FlutterTextInputView*)inputView {
-  if (![inputView isDescendantOfView:_inputHider]) {
-    [_inputHider addSubview:inputView];
-  }
+  // TODO: Find a different way to do what FlutterTextInputViewAccessibilityHider was doing
```

Comment:
The reason we have this weird dangling views are our effort to try to make inputView harder to be reach, yet is still reachable.

If you directly attach to keywindow, it will be very easy to focus in voiceover by swipes left and right. For example, if you activate the textfield in ios, swipe left to move the focus around, and you will see the textinputview receives the focus (It will be redirect back to the flutter textfield, but it is still bad user experience )

Reply:
I think I was just doing something wrong in the merge – it seems to work fine now as a child of the inputHider